HTML/html_completions.py

```diff
@@ -19,6 +21,15 @@ def make_completion(tag, completion=None):
         kind=(sublime.KIND_ID_MARKUP, 't', 'Tag')
     )
 
+def make_entity_completion(entity, rendered_char=None):
+    return sublime.CompletionItem(
+        trigger=entity,
+        completion=entity if entity.endswith(';') else f'{entity};',
+        completion_format=sublime.COMPLETION_FORMAT_SNIPPET,
```

Comment:
It should be safe to use `COMPLETION_FORMAT_TEXT` here.

Reply:
Is the implication of `COMPLETION_FORMAT_TEXT` that things like `${1:foo}` are inserted into the buffer verbatim?